### PR TITLE
KOGITO-548 - Fixed favicon

### PR DESCRIPTION
**Pull request by Sara4994**

### CODE REVIEW

File: packages/management-console/webpack.common.js [hunk 1]
```diff
@@ -11,7 +12,10 @@ module.exports = {
   plugins: [
     new HtmlWebpackPlugin({
       template: path.resolve(__dirname, 'src', 'index.html')
```
Comment (cristianonicolai) [inline packages/management-console/webpack.common.js]: @Sara4994 I found that all we need is to add the follwing:
```suggestion
      template: path.resolve(__dirname, 'src', 'index.html')
      favicon: 'src/favicon.ico'
    })
```

then rename the image to favicon.ico but keep it at the src level.
CopyWebpackPlugin will copy all images to image directory, which is not needed.

**Review by cristianonicolai — CHANGES_REQUESTED** [review]

File: packages/management-console/webpack.common.js (hunk 1 above)
Comment (Sara4994) [reply packages/management-console/webpack.common.js]: @cristianonicolai Yes i tried this at first place, but still favicon wasnt showing up. So went with other aproach which worked fine.. anyway will try onemore time.

Comment (cristianonicolai) [reply packages/management-console/webpack.common.js]: cool, ok, I got it working fine with this approach so let me know if doesnt work for you.

Comment (Sara4994) [reply packages/management-console/webpack.common.js]: @cristianonicolai Modified.

**Comment by cristianonicolai** [comment]
@Sara4994 I believe this PR could be closed.

**Pull request closed without merging**